docs/rebuild_admin.rst

```diff
@@ -0,0 +1,404 @@
+Rebuilding an Admin Workstation USB
```

Comment:
italicize Admin Workstation (we do it even in chapter headings, albeit inconsistently)

Reply:
fixed